README.md

```diff
@@ -20,65 +20,77 @@ Run this project as the root user:
 # bash ecs-logs-collector.sh
 ```
 
-Confirm if the tarball file was successfully created ( it can be .tgz or .tar.gz )
+Confirm if the tarball file was successfully created (it can be .tgz or .tar.gz)
 
 ```
-#ls collect.*
-collect.tgz
+#ls collect*
+collect-i-fffffffffffffffff.tgz
+
+collect:
+i-fffffffffffffffff
```

Comment:
does it matter that this is one f short of one below (in ecs-logs-collector.sh)

Reply:
Good catch :) . It does not matter but they are fixed now.